app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -53,7 +53,7 @@ banSiteConfirmation=Are you sure you want to delete this site?
 beta=beta
 bitcoin=Bitcoin
 bitcoinBalance=Please transfer:&nbsp;
-bitwarden=bitwarden&reg;
+bitwarden=Bitwarden&reg;
```

Comment:
Should we go ahead and use the same value as extensions.properties? Otherwise the name changes in preferences before and after enabling the Bitwarden extension. (Same for other extension strings in this file)

Reply:
I would suggest we do because then we can drop the &reg; part at the end...

Reply:
This is now fixed :+1: